### ISSUE TITLE
Smartpos-F.  Получение наличных и безналичных оплат перед закрытием смены

### ISSUE THREAD

**Issue by try2bajed1** [shtrih-m/javapos_shtrih#81]
При закрытии смены извлекаю сумму продаж
 Set_RegisterNumber(121)
checkResult(GetCashReg())
Get_ContentsOfCashRegister()

Можно ли каким-то образом получить сумму оплат наличными и безналичными?

**Comment by nyxiscoo1** [comment]
Да, нужно прочитать соответствующие регистры. В случае с безналичной оплатой их будет несколько, нужно будет просуммировать.
